ext/sqlite3/sqlite3.c

```diff
@@ -2040,14 +2040,10 @@ static int php_sqlite3_authorizer(void *autharg, int action, const char *arg1, c
 			if (memcmp(arg1, ":memory:", sizeof(":memory:")) && *arg1) {
 				if (strncmp(arg1, "file:", 5) == 0) {
 					/* starts with "file:" */
-					if (!arg1[5]) {
+					if (PG(open_basedir) && *PG(open_basedir)) {
```

Comment:
We can skip this test, as this is tested above already

Reply:
Ah, right. Thank you!